### PR TITLE
[feat]: Login Page Added for Ecom

### PR DESCRIPTION
**Pull request by raghav2110**
Hi, I have added login page for ecom. Please review my PR.

### CODE REVIEW

File: css/loginPage.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+.email-label{
+    display: block;
+    font-size: 1rem;
+    color: var(--headerbg);
+    font-weight: bold;
+    margin: 0.5rem 0;
+}
+
+.email-input{
+    border: 0.2rem solid #f3f4f6;
```
Comment (omgaikwad) [inline css/loginPage.css:53]: You can use color variable here.

**Review by omgaikwad — APPROVED** [review]
The code looks good.

File: pages/loginPage.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://splash-ui.vercel.app/CSS/main.css">
+    <link rel="stylesheet" href="/css/loginPage.css">
+    <title>Sneaker Street</title>
+</head>
+<body>
+    <header class="header-main">
+        <nav class="nav nav-dark" style="position: relative;">
```
Comment (Hamdrive) [inline pages/loginPage.html]: instead of inline styles, define your own custom CSS utility class and use it here

File: pages/loginPage.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        </nav>
+    </header>
+
+    <div class="main-wrapper">
+        <div class="login-wrapper">
+            <div class="login-text">Login</div>
+            <div class="input-area">
+                <form class="form-area">
+                    <div>
+                        <label class="email-label">Email</label>
```
Comment (Hamdrive) [inline pages/loginPage.html]: Consider using `for` attribute in `label` to link it to its `input` with `id` attribute

**Review by Hamdrive — CHANGES_REQUESTED** [review]
Overall good and clean code written. 
Have suggested to implement changes for label and input correlation, else all looks good!

File: pages/loginPage.html (hunk 3 above)
Comment (raghav2110) [reply pages/loginPage.html]: Got it will make that changes

File: pages/loginPage.html (hunk 2 above)
Comment (raghav2110) [reply pages/loginPage.html]: Okay will check that

**Review by Hamdrive — APPROVED** [review]
Changes made looks good, great practices followed
Approved and good to merge!

**Pull request merged**